packages/react-color-picker/rollup.config.js

```diff
@@ -10,7 +11,7 @@ export default {
       output: 'all-components.css',
     }),
     ts({
-      tsconfig: './tsconfig.build.json',
+      tsconfig: fileURLToPath(new URL('./tsconfig.build.json', import.meta.url)),
```

Comment:
🤔 originally thought this was an issue with Nx, but upon further investigation, Rollup is just really bad with `cwd`. I usually expect things to resolve relative to their declaring files, but Rollup inherently makes that hard to do; doesn’t seem to be unique to `@rollup/plugin-typescript`.